agents_infra/agents_infra/utils/communication.py

```diff
@@ -0,0 +1,32 @@
+from logtools import maybe_log_message
+from token_manager import TokenManager
```

Comment:
If these are local modules, consider using relative imports (e.g., `from .logtools import ...`) for package consistency.